roles/satellite-hostname/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: "check that variables are updated"
-  fail: msg="Please update the variables in roles/change_hostname/vars/main.yml"
+  fail: msg="Please update the variables in satellite-hostname-vars.yml"
```

Comment:
the satellite-hostname playbook is separate from the satellite-clone playbook, so they should just have to update `satellite-hostname-vars.yml`